Remnant: Changes to Return the Sample Timer & Hint (#4837)
endless-sky/endless-sky: data/remnant missions.txt

		fail

# "This mission is intended to serve as a notification that one timer has elapsed and how long to wait until the missions start again."
mission "Remnant: Return the Samples 3 Exploration Hint"
	name "Return the Samples Timer Elapsed"

> 	description "According to your commlink, Plume has reached a milestone in his research."

Reviewer: I feel like referring to "your commlink" is weird, probably since it's such a rarely used term in game. (It's used in the FW intro and a few times in the Remnant missions and that's it.) I'd suggest a change, but this mission actually always declines, so no need for a `name` or `description`.

Author: The description is there so that when <first> <last> uses their magical powers of prophecy to discern the true meaning of things, they can find it. (That's the best way I can think of to describe <first> <last>'s special ability, anyway. (along with their alternate ability - being able to rewind the past)

Reviewer: Ain't that what we have comments for? 🤔

Author: Well, comments are data outside the framework. Exceptions. If we have a built in way to include the same info in a format the game, players, and things like mission builders will all handle in a uniform, predictable, way. Why not use it?

Reviewer: I'm still not sure why you'd need `name` and `description` for a mission where they're never visible in game.

	landing

> 	source Aventine

Reviewer: ```suggestion
	source "Aventine"
```

	to offer
		has "event: remnant: return the samples timer"
	on offer
		event "remnant: wait for plume" 7 14
		conversation

> 			`As you disembark from <ship> you almost run into Plume as he strides across the landing area. "Captain <first>, I didn't see you there!" You glance back at the bulk of your ship that you just stepped out of and look at him curiously. His chant takes on a slightly apologetic tone "I've been lost in thought thinking about everything we have discovered lately, especially the most recent recordings you made. I still have a lot to analyze, though." He pauses, looking thoughtful. "That being said, I will probably have more work for you in a week or two."`

Reviewer: Why am I glancing back at my ship? Did he run past me and to it for some reason?
Missing period after "apologetic tone".
```suggestion
			`As you disembark from <ship> you almost run into Plume as he strides across the landing area. "Captain <first>, I didn't see you there!" You glance back at the bulk of your ship that you just stepped out of and look at him curiously. His chant takes on a slightly apologetic tone. "I've been lost in thought thinking about everything we have discovered lately, especially the most recent recordings you made. I still have a lot to analyze, though." He pauses, looking thoughtful. "That being said, I will probably have more work for you in a week or two."`
```

Author: I meant that comment about looking back at the ship along the lines of "You just landed and got off your ship, and plume almost runs into you; so you are looking back at your ship thinking "What, is my ship cloaked? Shouldn't it be obvious that I'm here?" 

But yeah, it doesn't come across really well.

			`	Plume asks a few more questions about what you have been doing, then politely excuses himself to continue his research. "Enjoy your explorations!" he trills to you as he departs.`

> 			fail

Reviewer: `fail` is not a conversation endpoint.
```suggestion
				decline
```

> 			`	Plume asks a few more questions about what you have been doing, then politely excuses himself to continue his research. "Please let me know if you visit Nenia again and find anything!" he trills to you as he departs.`

Reviewer 2: Maybe tone down the hint with something like this?
```suggestion
			`	Plume asks a few more questions about what you have been doing, then politely excuses himself to continue his research. "Please let me know if you uncover anything else about Nenia!" he trills to you as he departs.`
```

> event "remnant: wait for plume"

Reviewer: Should this be used anywhere, or is this for VS2?

Author: This is for VS2.
